src/constants/contractTokenAddresses.ts

```diff
@@ -53,7 +53,7 @@ import vust from 'assets/img/coins/vust.png';
 import vluna from 'assets/img/coins/vluna.png';
 
 import { isOnTestnet } from 'config';
-import contracts from './contracts';
+import contracts from '../utilities/contracts';
```

Comment:
I updated this file in the typing PR to be `contracts`. Which name would you like to use? You requested we drop addresses from the names of all the constants. I think it would make sense to also drop it from the name of the file.

Reply:
Ah yeah sorry for that, I think I made these changes before our discussion and forgot to make amendments. Let me update to what you've done in your PR instead